topics/proteomics/tutorials/metaproteomics/tutorial.md

```diff
@@ -96,12 +96,12 @@ The created dataset collection of the three *MGF files* in the history is used a
 >
 >    Section **Search Engine Options**:
 >
->    - **B-Search Engines**: `X!Tandem`
+>    - **Search Engines**: `X!Tandem`
 >
 >    > ### {% icon comment %} Comment
 >    >
->    > The section **Search Engine Options** contains a selection of sequence database searching
->    > programs that are available in SearchGUI. Any combination of these programs can be used for
+>    > This section **Search Engine Options** contains a selection of sequence database searching
```

Comment:
Any reason to replace `The`. It sounds better to me :) But I have usually no clue about english :)

Reply:
You're right, The does sound better!!!